cli/src/pcluster/config/cluster_config.py

```diff
@@ -1408,14 +1406,6 @@ def __init__(
         super().__init__(**kwargs)
         self.pools = pools
 
-    @property
```

Comment:
why are we removing this?

Reply:
just need check if dcv is enabled at individual pool level now since each pool has own managed security group